### PR TITLE
Add storagetransfer samples

### PR DESCRIPTION
**Pull request by shun-fan**

### CODE REVIEW

File: cloud-storage/storage-transfer/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+1. Set up a project on Google Developers Console.
+  1. Go to the [Developers Console](https://cloud.google.com/console) and create or select your project.
+     You will need the project ID later.
+1. Add the Storage Transfer service account, cloud-mobility@system.gserviceaccount.com as an 
+   editor of your project.
+1. Set up gcloud for application default credentials.
+  1. `gcloud components update`
+  1. `gcloud auth login`
+  1. `gcloud config set project PROJECT_ID`
+  1. `export GOOGLE_APPLICATION_CREDENTIALS=PATH/TO/CREDENTIALS.json`
```
Comment (lesv) [inline cloud-storage/storage-transfer/README.md]: Where did you get this from?

File: cloud-storage/storage-transfer/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+1. Install jar
+   `mvn install:install-file -Dfile=libstoragetransfer-v1-java-public.jar \`
+   `-DgroupId=com.google.storagetransfer.samples -DartifactId=libstoragetransfer -Dversion=1 -Dpackaging=jar`
+
```
Comment (lesv) [inline cloud-storage/storage-transfer/README.md]: How did I get this jar?

File: cloud-storage/storage-transfer/src/main/java/com/google/cloud/storage/storagetransfer/samples/AwsRequester.java [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+ */
+public final class AwsRequester {
+
+  private static final String JOB_DESC = "YOUR DESCRIPTION";
+  private static final String PROJECT_ID = "YOUR_PROJECT_ID";
+  private static final String AWS_SOURCE_NAME = "YOUR SOURCE BUCKET";
+  private static final String AWS_ACCESS_KEY_ID = "YOUR_ACCESS_KEY_ID";
+  private static final String AWS_SECRET_ACCESS_KEY = "YOUR_SECRET_ACCESS_KEY";
+  private static final String GCS_SINK_NAME = "YOUR_SINK_BUCKET";
+  private static final String START_DATE = "YYYY-MM-DD";
```
Comment (lesv) [inline cloud-storage/storage-transfer/src/main/java/com/google/cloud/storage/storagetransfer/samples/AwsRequester.java]: You might wish to have a line break as folks aren't going to be setting this?  Or if they are, why?

**Comment by shun-fan** [comment]
I also updated the pom to use Google checkstyle.

File: cloud-storage/storage-transfer/README.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+This app creates two types of transfers using the Transfer Service tool.
+
+## Prerequisites
+
+1. Set up a project on Google Developers Console.
+  1. Go to the [Developers Console](https://cloud.google.com/console) and create or select your project.
+     You will need the project ID later.
+1. Within Developers Console, select APIs & auth > Credentials.
+  1. Add a new JSON key credential for a service account.
```
Comment (lesv) [inline cloud-storage/storage-transfer/README.md]: Is this even close to how this is expressed on the page?

**Comment by lesv** [comment]
Just fix the readme, then LGTM

**Pull request merged**